### ISSUE TITLE
Typescript Error in node_modules/quasar/dist/types/index.d.ts

### ISSUE THREAD

**Issue by eyedean** [quasarframework/quasar#6094]
I just upgraded to 1.7.0 and I see some Typescript errors in my `node_modules/quasar/dist/types/index.d.ts`

Here is the error details:
```
node_modules/quasar/dist/types/index.d.ts:308:24 - error TS1005: ',' expected.

308     setDefaults ([index: string]: any): void
```
It also shows up in VSCode.
<img width="463" alt="Screenshot at Jan 13 04-32-17" src="https://user-images.githubusercontent.com/5694695/72256306-b6c9f000-35bd-11ea-8439-3d150318e051.png">

I looked into it and I found it's coming from https://github.com/quasarframework/quasar/blame/dev/ui/build/build.types.js#L69-L71 which has some activities recently.

As can be seen in [this playground](https://www.typescriptlang.org/play/#code/MYewdgzgLgBAtgUwhAhgcwQLhtATgSzDRgF4YByACwQBsaQYB3EXGgE3IG4AoUSEGggB09NAApEydAgCUPboSgJcAMxTAEMAB4wA3txiGYIANYB+bHkLEAPnpgBtQmwRbLUAkQC62FGACeMAC+MHYARiACCH4GRhAIUAAiCGoArjRQEDBiTmAubjge1j4wfv4y2ABuIPhssYbxSSko6ZkAjGIADrggnRDYAEoIoLhsADxWRAA0pQEAfBUw1bU8RjgJyWkZEABMXT192LqOzq7unmglZcGLy3VBQA) of Typescript, in order to specify the parameter of a function is an object with its keys being `string` either of the following works:
```
    setDefaults1(props: Record<string, any>): void;
    setDefaults2(props: { [index: string]: any }): void
```

which the generated .d.ts for LoadingBar has `    setDefaults ([index: string]: any): void` which is not working with typescript.

**Comment by pdanpdan** [comment]
Duplicate of https://github.com/quasarframework/quasar/issues/6088

**Comment by rstoenescu** [comment]
Fixed in 1.7.1.
